src/Pages/Recommend/RecommendTap.js

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from "react";
+import Recommend from "./RecommendComponent/Recommend";
+import RecommendReuest from "./RecommendComponent/RecommendRequest";
+import RecommendList from "./RecommendComponent/RecommendList";
+import "../Recommend/RecommendTap.scss";
+import styled from "styled-components";
+
+const obj = {
+  0: <Recommend />,
+  1: <RecommendReuest />,
+  2: <RecommendList />,
+  3: <Recommend />
+};
+
+export class RecommendTap extends Component {
+  constructor() {
+    super();
+    this.state = {
+      activeMenu: 0
+    };
+  }
+  clickHandler = menuId => {
+    this.setState({ activeMenu: menuId });
+  };
+
+  render() {
+    console.log(this.state.activeMenu);
+    return (
+      <RecommendPage>
+        <Main>
+          <Header>
+            <div>
+              <button
+                onClick={() => this.clickHandler(0)}
+                className={
+                  this.state.activeMenu === 0 ? "clickBtn" : "unClickBtn"
+                }
+              >
+                추천하기
+              </button>
+              <button
+                onClick={() => this.clickHandler(1)}
+                className={
+                  this.state.activeMenu === 1 ? "clickBtn" : "unClickBtn"
+                }
+              >
+                내가 받은 추천
+              </button>
+              <button
+                onClick={() => this.clickHandler(2)}
+                className={
+                  this.state.activeMenu === 2 ? "clickBtn" : "unClickBtn"
+                }
+              >
+                내가 한 추천
+              </button>
+              <button
+                onClick={() => this.clickHandler(3)}
+                className={
+                  this.state.activeMenu === 3 ? "clickBtn" : "unClickBtn"
+                }
+              >
+                설정
+              </button>
```

Comment:
반복되므로 map 사용해서 간단한게 줄일 수 있습니다!
```suggestion
     {["추천하기", "내가 받은 추천", "내가 한 추천", "설정"].map((content, index) => {
         return (
              <button
                onClick={() => this.clickHandler(index)}
                className={
                  this.state.activeMenu === index ? "clickBtn" : "unClickBtn"
                }
              >
               {content}
              </button>
              )}
           )}
```

Reply:
!!!!!!!!!!!!